code/game/objects/items/manuals.dm

```diff
@@ -80,7 +80,7 @@
 					<li>High-speed Drill</li>
 					</ul>
 				</li>
-				<li><b>Propulsion Device:</b> Powercell-powered electro-hydraulic system.</li>
+				<li><b>Propulsion Device:</b> Power cell-powered electro-hydraulic system.</li>
```

Comment:
...This could probably use rewording regardless of your change.

Reply:
Thought about this one